packages/kit/src/components/AccountSelector/NetworkSelectorTrigger/NetworkSelectorTrigger.tsx

```
@@ -211,3 +212,87 @@ export function ControlledNetworkSelectorTrigger({
+    disabled,
+    openChainSelector,
+    title,
+    selectorNetworks,
+    current?.id,
+    onChange,
+  ]);
+  return (
+    <XStack
+      testID="account-network-trigger-button"
+      role="button"
+      flexShrink={1}
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)